modules/executors/variables.tf

```diff
@@ -31,7 +31,7 @@ variable "machine_image" {
 
 variable "machine_type" {
   type        = string
-  default     = "n1-standard-4" // 4 vCPU, 15GB
+  default     = "c2-standard-8" // 8 vCPU, 32GB
```

Comment:
this was completely misconfigured, 2 jobs with 4 CPU each and a machine with 8 total 😆